contracts/script/GenerateAlloc.s.sol

```diff
@@ -49,87 +50,69 @@ contract GenerateAlloc is Script {
 
     string internal dumpPath = getDumpPath();
     bool public saveState = true;
-    uint256 public constant STORY_CHAIN_ID = 1415; // Story mainnet chain ID. TBD
     // Optionally allocate 10k test accounts for devnets/testnets
     bool private constant ALLOCATE_10K_TEST_ACCOUNTS = false;
     // Optionally keep the timelock admin role for testnets
     bool private constant KEEP_TIMELOCK_ADMIN_ROLE = true;
 
     /// @notice this call should only be available from Test.sol, for speed
     function disableStateDump() external {
-        require(block.chainid == 31337, "Only for local tests");
+        require(block.chainid == ChainIds.FOUNDRY, "Only for local tests");
         saveState = false;
     }
 
     /// @dev this call should only be available from Test.sol
     function setAdminAddresses(address protocol, address executor, address guardian) external {
```

Comment:
FWIW, we could write to internal storage slots directly in tests (but having this function in script doesn't impact performance or anything).